dask/array/core.py

```diff
@@ -1151,7 +1152,7 @@ def store(
 
     if return_stored:
         store_dsk = HighLevelGraph(layers, dependencies)
-        load_store_dsk = store_dsk
+        load_store_dsk: HighLevelGraph | Mapping = store_dsk
```

Comment:
```suggestion
        load_store_dsk: HighLevelGraph | dict = store_dsk
```
Avoid unnecessary abstraction